src/components/_datepicker-demo/CustomInput/CustomInput.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCalendar, faClock } from "@fortawesome/free-solid-svg-icons";
+
+class CustomInput extends React.PureComponent {
+  render() {
+    const icons = {
+      calendar: faCalendar,
+      clock: faClock
+    };
+    const selectedIcon = icons[this.props.icon];
+    return (
+      <div
```

Comment:
Let's avoid inline styling.

Reply:
i used quick and dirty inline styling to put together a quick demo for the component. i assume the demo is not gonna be merged into the project in the end